app/src/main/java/com/team22/soundary/feature/main/data/ShareRepositoryImpl.kt

```diff
@@ -0,0 +1,83 @@
+package com.team22.soundary.feature.main.data
+
+import android.net.Uri
+import com.team22.soundary.feature.main.domain.Share
+import com.team22.soundary.feature.main.domain.ShareRepository
+import com.team22.soundary.feature.main.domain.Song
+import com.team22.soundary.feature.main.domain.User
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import java.util.Date
+
+class ShareRepositoryImpl : ShareRepository {
+    private var mockDataList: MutableList<ShareEntity> = mutableListOf(
+        ShareEntity(
+            UserEntity("", "춘식이", "", "", "", Date(), Date()),
+            SongEntity("", "스토커", "", "", "10cm", Date()),
+            "이거듣고 눈물흘림",
+            false
+        ),
+        ShareEntity(
+            UserEntity("", "춘식이", "", "", "", Date(), Date()),
+            SongEntity("", "Celebrity", "", "", "IU", Date()),
+            "좋은 노래네요",
+            false
+        ),
+        ShareEntity(
+            UserEntity("", "라이언", "", "", "", Date(), Date()),
+            SongEntity("", "Dynamite", "", "", "BTS", Date()),
+            "신나는 노래 추천!",
+            false
+        ),
+        ShareEntity(
+            UserEntity("", "춘식이", "", "", "", Date(), Date()),
+            SongEntity("", "Tokyo Flash", "", "", "Vaundy", Date()),
+            "노래입니다.",
+            false
+        ),
+        ShareEntity(
+            UserEntity("", "콘", "", "", "", Date(), Date()),
+            SongEntity("", "Supernova", "", "", "aespa", Date()),
+            "내 최애 노래야!",
+            true
+        ),
+    )
+
+
+    override fun getShareList(): Flow<List<Share>> = flow {
+        emit(
+            mockDataList.map { it.toVO() }
+        )
+    }
```

Comment:
p999)  아래와 같은 형태로 치환이 가능합니다 :)
```suggestion
    override fun getShareList(): Flow<List<Share>> = flowOf(mockDataList).map { it.toVO() }
```

Reply:
제안 주신대로 수정하였습니다. 훨씬 깔끔하니 좋네요!